.changelog/1762113734.md

```diff
@@ -0,0 +1,14 @@
+---
+applies_to:
+- server
+authors:
+- drganjoo
+references:
+- smithy-rs#3362
+breaking: false
+new_feature: true
+bug_fix: false
+---
+Add http-1x codegen support for server SDK generation
+
+    Enables generation of server SDKs compatible with either http@0.2.x/hyper@0.x or http@1.x/hyper@1.x based on the http-1x codegen flag.
```

Comment:
need more detail in the changelog message--e.g. changelog needs to say what the actual flag is, how to enable it etc.